Dependency/peg-markdown-highlight/HGMarkdownHighlighter.m

```diff
@@ -101,15 +99,18 @@ - (instancetype) initWithTextView:(NSTextView *)textView
 
 #pragma mark -
 
+- (dispatch_queue_t)parsingQueue
+{
+    if (_parsingQueue == nil)
+    _parsingQueue = dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0);
```

Comment:
Depends on what "better" means. I am not that fluent with GCD intricacies :) I picked that because it results in a concurrent instead of a serial queue by default and is pretty straightforward. Right now, the system manages what'll happen. I'm fine with that until we run into trouble, though increasing the priority to "high" might be a good idea. I do not know if a custom queue will increase the speed of executing enqueued blocks -- or if that kind of optimization can affect this kind of code at all.